dpbench/infrastructure/benchmark.py

```diff
@@ -853,8 +857,20 @@ def get_data(self, preset: str = "L") -> Dict[str, Any]:
         for k, v in parameters.items():
             data[k] = v
 
+        # 4. Store types of selected precision dict in "data" dict.
+        if self.info.init.types_dict != "":
+            types_dict = dict()
+            for k, v in cfg.GLOBAL.dtypes.items():
```

Comment:
```suggestion
            for _type, precision_strings in cfg.GLOBAL.dtypes.items():
```
May be something more specific like this? It would be easier to read the code.

Reply:
Done